javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+const testProductNames = ["Flat Screen", "Mobile Phone", "Wallet"];
+
+function renderProducts(products) {
```

Comment:
this function is doing everything. You should have a clarity in your head on what goes into the DOM and what can be done without the DOM. Maybe it's not clear why yet, but you'll find it useful later. See this: https://www.youtube.com/watch?v=fa8eUcu30Lw